src/components/Root/FFetch.js

```diff
@@ -248,7 +251,7 @@ export class FFetch {
       }
 
       // AT is valid or unnecessary; execute the fetch
-      if (this.isPermissibleRequest(resource, this.tokenExpiration, okapi.url)) {
+      if (rtrIgnore || this.isPermissibleRequest(resource, this.tokenExpiration, okapi.url)) {
```

Comment:
it's the easiest fix to make SSO work as currently request to fetch permissions and after set up okapi session is blocked https://github.com/folio-org/stripes-core/pull/1415/files#diff-bb2e7a2fa6dcb390a09c12b8e1ee5c0f3354c479d0e537252e7f6328a029d0a6R266

https://github.com/folio-org/stripes-core/pull/1415/files#diff-c9d695b1d57404dfed01196c8cf479c0e4cd48583f6c78e7db23402582846864R751

Reply:
we can create tokenExp in okapi session before permissions request, but after it will create send refresh request anyway as createOkapiSession parses response - as for me it's better to not initiate partial okapi session before perms request